lib/utils.bash

```diff
@@ -48,7 +48,7 @@ install_version() {
   local install_path="$3"
   local os_distribution="$(uname -s)"
   local tool_cmd="$(echo "aws --help" | cut -d' ' -f1)"
-  local test_path="${install_path}/${tool_cmd}"
+  local test_path="${install_path}/bin/${tool_cmd}"
```

Comment:
This was just a side benefit of changing the Mac v2 bin-path: Now all installs share the same `test_path`.